### PR TITLE
code and readme updates

### PR DESCRIPTION
**Pull request by erikarasnick**

### CODE REVIEW

**Comment by erikarasnick** [comment]
also closes #13

File: entrypoint.R [hunk 1]
```diff
@@ -1,30 +1,26 @@
...
-              description = 'links geocoded coordinates with date ranges to cooresponding census tracts from the appropriate vintage')
+dht::greeting()
+
+## load libraries without messages or warnings
+withr::with_message_sink("/dev/null", library(dplyr))
+withr::with_message_sink("/dev/null", library(tidyr))
+withr::with_message_sink("/dev/null", library(sf))
 
 old_warn <- getOption("warn")
 options(warn = -1)
```
Comment (cole-brokamp) [inline entrypoint.R:11]: is this still necessary?

File: entrypoint.R [hunk 2]
```diff
@@ -1,30 +1,26 @@
...
+      "
 
 opt <- docopt::docopt(doc)
+
 ## for interactive testing
 ## opt <- docopt::docopt(doc, args = 'test/my_address_file_geocoded.csv')
 
-message('reading input file...')
+message("reading input file...")
 raw_data <- readr::read_csv(opt$filename)
```
Comment (cole-brokamp) [inline entrypoint.R:24]: we should use `dht::read_lat_lon_file()` for this?

**Review by cole-brokamp — CHANGES_REQUESTED** [review]

File: entrypoint.R (hunk 1 above)
Comment (erikarasnick) [reply entrypoint.R]: This was to silence package load warnings, right? No, I don't think we need it.

**Review by cole-brokamp — APPROVED** [review]

**Pull request merged**